plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestingThriftHiveMetastoreBuilder.java

```diff
@@ -18,6 +18,7 @@
 import io.trino.filesystem.TrinoFileSystemFactory;
 import io.trino.filesystem.hdfs.HdfsFileSystemFactory;
 import io.trino.plugin.hive.metastore.HiveMetastoreConfig;
```

Comment:
`Add testing framework for ThriftHiveMetastore` 

You could provide a more descriptive name here.

e.g. `Test hive metastore operations involving retries`